pdns/dnsdistdist/doh.cc

```diff
@@ -470,6 +574,13 @@ try
 
   string path(req->path.base, req->path.len);
 
+  for (const auto& entry : dsc->df->d_responsesMap) {
+    if (entry->matches(path)) {
+      handleResponse(*dsc->df, req, entry->getStatusCode(), entry->getContent(), std::string());
```

Comment:
`Content-type` header will be set by default depending on the desired status code.
This would be `application/dns-message` in case of a 200  response which may not be what we want.